packages/consent-manager/src/integration-helpers.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { hex } from 'wcag-contrast'
+
+export function getForegroundColor(bgHex: string): string {
+  return hex(bgHex, '#000') >= 7 ? '#000' : '#fff'
+}
+
+export interface IconComponentProps {
```

Comment:
We should this move this somewhere closer to the config where it's used. And potentially export in index as well, as we want people to easily import and use this type to make sure their components adhere to our interface.